README.md

```diff
@@ -52,12 +52,13 @@ If you are running an older version (for example, if you originally installed WS
 
 - `sudo do-release-upgrade` ... this updates the version of Ubuntu. _Please note_, this will likely disable any external package sources, such as the one added when installing Mono.
 - Once you're done with that, then you just need to update `apt-get`
-    - Add back Mono's package repository source, as described in [the download instructions](www.mono-project.com/download/#download-lin-ubuntu).
-    - `sudo apt-get update`
-    - `sudo apt-get upgrade`
-    - `sudo apt-get dist-upgrade`
+- Steps to set up the local WSL mdoc environment
+    - Install make `sudo apt-get install make`
+    - Install mono, Add back Mono's package repository source, as described in [the download instructions](www.mono-project.com/download/#download-lin-ubuntu).
```

Comment:
In preview page, this link is 404 not-found.
I think here this link should be "http://....". Otherwise, the Github will parse this URL to append to the git repo link. Like "
https://github.com/mono/api-doc-tools/blob/431bc7aee5c5ad25785a72375d1dab1f19ba2778/www.mono-project.com/download/#download-lin-ubuntu"